debian/control

```diff
@@ -9,6 +9,7 @@ Build-Depends: bash-completion,
                dh-systemd | debhelper (>= 13.3),
                gettext,
                git,
+               golang (>=1.6),
```

Comment:
This will probably break the trusty package, as golang there is 1.6.

Just to confirm, do we need version 1.6 specifically on 1.3 (which is present in trusty)  would work too ?

Reply:
I removed the version constraint and it seems to be building fine on trusty
There seems to be an unrelated building error on focal though which I'm trying to recreate now